src/components/navbar/navbar.component.tsx

```diff
@@ -88,7 +94,11 @@ const Navbar: React.FC<NavbarProps> = ({
                   hidePanel();
                 }}
               >
-                <Logo />
+                {config.logo.name ? (
+                  config.logo.name
+                ) : (
+                  <Logo src={config.logo?.src} alt={config.logo?.alt} />
```

Comment:
Here we say logo may be undefined or null and above we assume its never null and check `config.logo.name`. Something is not right.